library/src/main/res/values-v9/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright 2012 Square, Inc. -->
 
-<resources>
-  <string name="month_name_format">LLLL yyyy</string>
+<resources xmlns:tools="http://schemas.android.com/tools">
+  <string name="month_name_format" tools:ignore="MissingTranslation">LLLL yyyy</string>
```

Comment:
Sure; that would work too; done.

On Mon, Aug 31, 2015 at 11:48 PM, Jake Wharton <notifications@github.com>
wrote:

> In library/src/main/res/values-v9/strings.xml
> <https://github.com/square/android-times-square/pull/231#discussion_r38363777>
> :
>
> > @@ -1,6 +1,6 @@
> >  <?xml version="1.0" encoding="utf-8"?>
> >  <!-- Copyright 2012 Square, Inc. -->
> >
> > -<resources>
> > -  <string name="month_name_format">LLLL yyyy</string>
> > +<resources xmlns:tools="http://schemas.android.com/tools">
> > +  <string name="month_name_format" tools:ignore="MissingTranslation">LLLL yyyy</string>
>
> I think I would prefer if you reverted this and downgraded the check to a
> warning in the lintOptions of the build.gradle instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/android-times-square/pull/231/files#r38363777>.
>